tests/test_scripts_pythondistdeps.py

```diff
@@ -0,0 +1,238 @@
+                                from pkg_resources import parse_version
+                                abi = f"cp{py_version_nodots}"
+                                # The "m" was removed from the abi flag in Python version 3.8
+                                if parse_version(py_version) < parse_version('3.8'):
+                                    abi += "m"
+
+                                # Install = download and unpack wheel into our
+                                #   temporary directory
+                                sys.argv[1:] = ["install", "--no-deps",
+                                        "--only-binary", ":all:",
+                                        "--platform", "manylinux1_x86_64",
+                                        "--python-version", py_version,
+                                        "--implementation", "cp",
+                                        "--abi", abi,
+                                        "--target", temp_dir,
+                                        "--no-build-isolation",
+                                        f"{package}=={pkg_version}"]
+                            else:
+                                # Download sdist that we'll unpack later
+                                sys.argv[1:] = ["download", "--no-deps",
+                                        "--no-binary", ":all:",
+                                        "--dest", temp_dir,
+                                        "--no-build-isolation",
+                                        f"{package}=={pkg_version}"]
+
+                            try:
+                                # run_module() alters sys.modules and sys.argv, but restores them at exit
+                                runpy.run_module("pip", run_name="__main__", alter_sys=True)
+                            except SystemExit as exc:
+                                pass
+                            finally:
+                                sys.argv[:] = backup_argv
+
+                            temp_path = Path(temp_dir)
+                            if type == "sdist":
+                                # Wheel were already unpacked by pip, sdists we
+                                # have to unpack ourselves
+                                sdist_path = next(temp_path.glob(f"{package}-{pkg_version}.*"))
+
+                                if sdist_path.suffix == ".zip":
+                                    import zipfile
+                                    archive = zipfile.ZipFile(sdist_path)
+                                else:
+                                    import tarfile
+                                    archive = tarfile.open(sdist_path)
+
+                                archive.extractall(temp_path)
+                            try:
+                                info_path = next(temp_path.glob(f"**/*{suffix}"))
+
+                                # pro wheel pojdme assertovat ze info path se rovna install path
```

Comment:
English? :D

Reply:
Omg, that was just my internal note from our review, not something I wanted to keep. Oops :D